src/test/java/lotto/domain/LottoNumberTest.java

```diff
@@ -0,0 +1,55 @@
+package lotto.domain;
+
+import lotto.domain.exception.NotValidLottoNumberException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class LottoNumberTest {
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3, 4, 5, 6, 10})
+    public void createLottoNumber(int number) {
+        //Given & When
+        LottoNumber lottoNumber = LottoNumber.from(number);
+
+        //Then
+        assertThat(lottoNumber).isEqualTo(LottoNumber.from(number));
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {-1, 0, 46, 50})
+    public void notValidLottoNumber(int number) {
+        assertThatThrownBy(() ->
+                LottoNumber.from(number)
+        ).isInstanceOf(NotValidLottoNumberException.class);
+    }
+
+    @Test
+    public void findAllLottoNumbersMapToList() {
+        //Given & When
+        List<Integer> result = LottoNumber.findAllLottoNumbers();
+
+        //Then
+        assertThat(result).isNotNull();
+        assertThat(result).hasSize(45);
```

Comment:
```suggestion
        assertThat(result).hasSize(LottoNumber.MAX_LOTTO_NUMBER);
```
로또 번호 갯수가 변경되면 해당 테스트는 실패 하게 되네요
상수를 이용해보면 어떨까요 ? 😄